modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/converter/DLVideoFFMPEGVideoConverter.java

```diff
@@ -66,7 +66,7 @@ public InputStream generateVideoPreview(File file, String containerType)
 					_getVideoBitRate(videoProperties, containerType)),
 				"-vf",
 				String.format(
-					"scale=min(%d\\,iw):-2",
+					"\"scale=min(%d\\,iw):-2\"",
```

Comment:
Were you able to reproduce the original issue locally? For what I know, `ProcessBuilder` does not spawn a shell to run the command, so this will fail unless FFMPEG is interpreting the quotes itself.

Reply:
based on an example given on [their site](https://trac.ffmpeg.org/wiki/Scaling)
`ffmpeg -i "${In}" -vf "scale=320:240,setsar=1:1" "${Out}"`
I've tried using the quotes with both a Windows console and a gitBash to simulate Linux

Reply:
But as ProcessBuilder doesn't invoke a shell (which is the thing that processes quotes, does variable replacement, etc.), the quotes will do nothing (in the best case), and will probably make ffmpeg choke on the argument unless it handles the quotes itself.